packages/markdown/__tests__/__snapshots__/components.test.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Components Callout 1`] = `"<blockquote class=\\"callout callout_error\\" theme=\\"❗️\\"><h3 class=\\"callout-heading false\\"><span class=\\"callout-icon\\">❗️</span><p>Error Callout</p></h3><p>Lorem ipsum dolor.</p></blockquote>"`;
+exports[`Components Callout 1`] = `"<blockquote class=\\"callout callout_error\\" theme=\\"❗\\"><h3 class=\\"callout-heading false\\"><span class=\\"callout-icon\\">❗</span><p>️ Error Callout</p></h3><p>Lorem ipsum dolor.</p></blockquote>"`;
```

Comment:
Should there be a space at the top of the `<p>`, or between the emoji and paragraph?

Reply:
It's because the unicode for the ❗️emoji seems to include a trailing space to regex, hence the extra preceding space. But other than IE, this doesn't effect the actual render.